app/uk/gov/hmrc/merchandiseinbaggage/service/CalculationService.scala

```diff
@@ -33,6 +35,21 @@ class CalculationService @Inject()(mibConnector: MibConnector)(implicit ec: Exec
   def paymentCalculations(importGoods: Seq[ImportGoods])(implicit hc: HeaderCarrier): Future[CalculationResults] =
     mibConnector.calculatePayments(importGoods.map(_.calculationRequest)).map(withLogging)
 
+  def thresholdCheck(declarationJourney: DeclarationJourney)(implicit hc: HeaderCarrier): OptionT[Future, Boolean] =
+    if (declarationJourney.amendmentIfRequiredAndComplete.isDefined)
```

Comment:
There is a method on declarationJourney that does this. `declarationJourney.amendmentRequiredAndComplete`